package.json

```diff
@@ -18,41 +18,52 @@
     }
   ],
   "scripts": {
-    "build": "bower install && gulp",
-    "jshint": "gulp jshint",
-    "jscs": "jscs gulpfile.js assets/scripts/*.js"
+    "build:production": "export SAGE_ENV=production && webpack",
+    "build": "export SAGE_ENV=development && webpack",
+    "watch": "export SAGE_ENV=development && node dev.js"
```

Comment:
I made two more [commit](https://github.com/roots/sage/compare/sage-9...ptrckvzn:webpack-sage-9?expand=1#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R21) on my branch. I removed the variables for two of the scripts. I can  work to remove it from the watch script too